iotedged/src/main.rs

```diff
@@ -378,6 +381,18 @@ struct Options {
 	#[structopt(long)]
 	device_id: String,
 
+	/// `keyd` API endpoint
+	#[structopt(long, default_value = "unix:///var/lib/aziot/keyd.sock")]
+	keyd_url: url::Url,
```

Comment:
To fit with the naming scheme, these are "endpoints" (as in the config files) or "URIs" (as in the code), not "URLs".